### PR TITLE
stopwatch was added

### PR DESCRIPTION
**Pull request by Anastasiia-Svintsova**
- [DEMO LINK](https://Anastasiia-Svintsova.github.io/layout_stop-watch/)

### CODE REVIEW

**Review by SeKaM0 — APPROVED** [review]
Very cool. Nice job

File: src/styles/main.scss [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+  &__clock {
+    position: absolute;
+    top: 100px;
+  }
+}
+
+.clock {
+  width: 300px;
+  height: 300px;
```
Comment (Olexandr6) [inline src/styles/main.scss:90]: Create a var, for example: `$clock-size`

File: src/styles/main.scss [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.stopwatch {
```
Comment (Olexandr6) [inline src/styles/main.scss:35]: Create a separate file per each BEM block

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
Almost done🔥, check comments👇

File: src/styles/blocks/stopwatch.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      content: "PAUSE-OFF";
+    }
+  }
+
+  &__button-on:checked ~ &__clock::after {
+    animation: move 60s linear infinite forwards;
+  }
+
+  &__button-on:checked ~ &__clock::before {
+    animation: move 1s linear infinite forwards;
```
Comment (illiarerun) [inline src/styles/blocks/stopwatch.scss:37]: seconds-hand should do 1 full rotation in 60s, not in 1 second

File: src/index.html [hunk 4]
```diff
@@ -1,14 +1,52 @@
...
+        type="checkbox"
+        name="On"
+        class="stopwatch__button-on"
+        id="button-on"
+      >
+
+      <label
+        for="button-on"
+        class="stopwatch__button-on-label"
+      ></label>
```
Comment (illiarerun) [inline src/index.html:23]: put input inside label to improve readability

File: src/styles/blocks/clock.scss [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  }
+
+  &__hor-numbers {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    gap: 80%;
+    justify-content: center;
+    align-items: center;
```
Comment (illiarerun) [inline src/styles/blocks/clock.scss:38]: you have a lot of repeated styles here, move them to template or mixin

File: src/styles/blocks/clock.scss [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+    font-size: 20px;
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (illiarerun) [inline src/styles/blocks/clock.scss:41]: set font-family for the whole page, use * selector

File: src/styles/blocks/stopwatch.scss [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+      content: "PAUSE-ON";
+    }
+
+    &:checked ~ &-label::after {
+      content: "PAUSE-OFF";
+    }
+  }
+
+  &__button-on:checked ~ &__clock::after {
+    animation: move 60s linear infinite forwards;
```
Comment (illiarerun) [inline src/styles/blocks/stopwatch.scss:33]: minutess-hand should do 1 full rotation in 3600s, not in 60 seconds

File: src/styles/utils/_extends.scss [hunk 8]
```diff
@@ -0,0 +1,23 @@
+%stopwatch-hand {
+  box-sizing: border-box;
```
Comment (illiarerun) [inline src/styles/utils/_extends.scss:2]: box-sizing should be the same for all elements, use * selector for it

**Review by illiarerun — CHANGES_REQUESTED** [review]
Overall great job 🔥 But you should fix these little mistakes

File: src/index.html (hunk 4 above)
Comment (Anastasiia-Svintsova) [reply src/index.html:23]: I wanted to do this, but in that case, I can't set up the animation when the button is checked.

**Review by AlinaMatuschak — APPROVED** [review]
WoW. Great work 🤩

**Pull request closed without merging**